drivers/scheduler/k8s/specs/rabbitmq/px-rabbitmq-app.yml

```diff
@@ -0,0 +1,210 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: endpoint-reader
+rules:
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["get"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: endpoint-reader
+subjects:
+  - kind: ServiceAccount
+    name: rabbitmq
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: endpoint-reader
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rabbitmq
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: rabbitmq-internal
+  labels:
+    app: rabbitmq
+spec:
+  clusterIP: None
+  ports:
+    - name: http
+      protocol: TCP
+      port: 15672
+    - name: amqp
+      protocol: TCP
+      port: 5672
+  selector:
+    app: rabbitmq
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: rabbitmq
+  labels:
+    app: rabbitmq
+    type: LoadBalancer
```

Comment:
we cannot create type LoadBalancer..

this is going to create an external IP on each deploy on cloud runs.

External load balancer is required only if someone wants to connect to this rabiitmq instance from outside the cluster. Just type `ClusterIP` should suffice here.

Reply:
it's just a label, to actually create a load balancer we need to add under spec

Reply:
we need a separated service which is used by IO sidecar since we cannot set session affinity to rabbitmq communication network which we called `internal`

Reply:
```
 thiago@Thiagos-MacBook-Pro   ~  kubectl --kubeconfig kubeconfig describe svc rabbitmq
Name:              rabbitmq
Namespace:         default
Labels:            app=rabbitmq
                   type=LoadBalancer
Annotations:       <none>
Selector:          app=rabbitmq
Type:              ClusterIP
IP:                10.233.42.136
Port:              rabbitmq-mgmt-port  15672/TCP
TargetPort:        15672/TCP
Endpoints:         10.233.107.70:15672,10.233.108.60:15672,10.233.124.72:15672 + 1 more...
Port:              rabbitmq-amqp-port  5672/TCP
TargetPort:        5672/TCP
Endpoints:         10.233.107.70:5672,10.233.108.60:5672,10.233.124.72:5672 + 1 more...
Session Affinity:  ClientIP
Events:            <none>
```